RELEASING.md

```diff
@@ -32,3 +32,10 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 ## Releasing
 
 The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [MAINTAINERS](MAINTAINERS.md).
+
+1. Create a tag, e.g. 1.0.0, and push it to this GitHub repository.
+1. The [release-drafter.yml](.github/workflows/release-drafter.yml) will be automatically kicked off and is responsible for drafting a new release on GitHub.
+1. Before creating a draft release, this workflow creates a GitHub issue asking for approval from the [maintainers](MAINTAINERS.md). See sample [issue](https://github.com/gaiksaya/opensearch-net/issues/10). The maintainers need to approve in order to continue the workflow run.
+1. This draft release triggers the [jenkins release workflow](https://build.ci.opensearch.org/job/opensearch-net-release) as a result of which the client is released on [Nuget.org](https://www.nuget.org/profiles/opensearchproject). Please note that the release workflow is triggered only if created release is in draft state.
+1. Once the above release workflow is successful, the drafted release on GitHub is published automatically.
+1. Increment the version to next iteration. See [example](hhttps://github.com/opensearch-project/opensearch-net/pull/93).
```

Comment:
Theres a typo in one of the links `hhttps` rather than `https`, also could you please number these items correctly in the markdown.
```suggestion
1. Increment the version to next iteration. See [example](https://github.com/opensearch-project/opensearch-net/pull/93).
```

Reply:
Fixed. Also the items are ordered list in markdown already. This way we don't have to care about numbering. Markdown takes care of it. Try seeing it in [preview](https://github.com/opensearch-project/opensearch-net/blob/8c3bc8f12f1d733ee2ec84bc55e09d28bca3afa0/RELEASING.md).

Reply:
> Fixed. Also the items are ordered list in markdown already. This way we don't have to care about numbering. Markdown takes care of it. Try seeing it in [preview](https://github.com/opensearch-project/opensearch-net/blob/8c3bc8f12f1d733ee2ec84bc55e09d28bca3afa0/RELEASING.md).

I realise Markdown will auto-number the items, just personal preference for readability of the plain-text copy.

Reply:
Fixed!